src/main/java/jaz/Zer.java

```diff
@@ -95,7 +104,7 @@ public Zer(byte sanitizer) {
   // choose a parameterless method to keep the string that the sanitizer guides the fuzzer to
   // generate within the 64-byte boundary required by the corresponding guiding methods.
   public static void el() {
-    if (isSanitizerEnabled(EXPRESSION_LANGUAGE_SANITIZER_ID)) {
+    if (isSanitizerEnabled(REFLECTIVE_CALL_SANITIZER_ID)) {
```

Comment:
Could you add a comment here describing why we are using `REFLECTIVE_CALL_SANITIZER_ID`? This could avoid confusion in the future.